docs/website/docs/developers/performance/profiling-with-tracy.md

```diff
@@ -176,6 +179,9 @@ is wanted, set the CMake setting
 passing that `--iree-llvmcpu-link-embedded=false` when compiling test/benchmark
 modules.
 
+TODO: now that `--iree-hal-dump-executable-sources-to=...` is also
+required, we probably need that too in IREE's own test and benchmark suites.
```

Comment:
Let's drop this TODO. If we dumped sources on CI then any traces the CI produced wouldn't be portable anyway. Better to not spread the non-portable dev flag more. If we embed sources into .vmfb files / traces then we can enable _that_.